src/main/webapp/scripts/redirect-page-script.js

```diff
@@ -0,0 +1,23 @@
+var roomId;
+const privateRoomPath = "/views/private-room.html?id=";
+const joinRoomPath = "/views/join-room.html";
+
+// Send inputted room Id to VerifyRoomServlet them redirect them to proper page
+async function verifyRoom() {
+    roomId = getRoomId(window.location.href);
```

Comment:
var roomId. Having roomId as a global variable can cause issues if this is 'multi-threaded' (even though website javascript isn't, it can behave like it is)